ts/api/api.ts

```diff
@@ -325,6 +331,14 @@ class OpenYoloApiImpl implements OpenYoloApi {
         this.areTimeoutsDisabled ? undefined : DEFAULT_TIMEOUTS.proxyLogin;
     return request.dispatch(credential, timeoutMs);
   }
+
+  async cancelLastOperation() {
```

Comment:
Are you talking about the tests or something else ? The wrapped browser APIs are in `navigator_credentials.ts` which is already been implemented.